balance: Убирает требование двух рук и разброс при использовании голосигары

vovakons opened the pull request:
## Что этот ПР делает

Убирает требование двух рук и разбросы/отдачу при использовании голосигары.
Повышает цену голосигары с 10 тк до 35 тк.

## Почему это хорошо для игры

Роден и меланхолик попросили сделать, за баланс я не отвечаю.

## Демонстрация изменений

https://github.com/user-attachments/assets/91c6afbd-f9b5-4571-8a3a-f8b18d34db4e


## Тестирование

Проверил все на локалке, вроде ок.

vovakons committed:
holo cigar no spread

Трейт badass теперь убирает все разлеты пуль.
Немного снижены цены в тк одиночных модулей в аплинке.

vovakons committed:
disable recoil for badass trait

Отключение отдачи для трейта badass

vovakons committed:
remove two hands need if badass trait

Убирает требование двух рук для тяжелого оружия если у пользователя есть
трейт badass.

vovakons committed:
holo cigar tk price

Syvexer commented:
Круто, а нахуя цена 35?

vovakons commented:
> Круто, а нахуя цена 35?

Наверное потому что итем стал слишком сильным для 10 ТК?
Пара втшек дадут 60 выстрелов без разброса и позволяют убивать за полторы секунды зажима.

tsoivecen199015 commented:
Слишком хуйня > Слишком имба.

Ранее голосигары не покупали потому что мало пользы и дорого стоит

Теперь голосигару не будут покупать потому что дорого стоит и за 35 ТК можно накупить других крутых вещей вероятно.

Так же стоит учитывать что сама по себе голосигара бесполезна, и раскрывает себя она только тогда когда у агента появляется оружие

Просто в сравнение 
Командос что даёт ножевой бой, оружие на 20 урона + кучу слабеньких метательных ножей  33 ТК.
Силовой кулак что двумя ударами вводит в крит 18 ТК.
Стечкин с зажигательными 25 ТК.

vovakons commented:
> Слишком хуйня > Слишком имба.
> Баланс не найден

Как забанить Ивана в гите? 🥹

Syvexer commented:
> > Круто, а нахуя цена 35?
> 
> Наверное потому что итем стал слишком сильным для 10 ТК? Пара втшек дадут 60 выстрелов без разброса и позволяют убивать за полторы секунды зажима.

Какая то ебаная логика, пусть тогда с ней 2 втшки поставляются в бандл, лол. Предмет который нихуя по своей сущности не делает стоит треть аплинка по причине : "ну с вт слишком имба" блять звучит даже глупо. Автофаера у станционных пушек просто не должно быть, это для обр и нюки фишка, ну по моему мнению.

Syvexer commented:
Предмет задумывался как рофло штука для акимбо за пару тк, а не как хуйнюшка на треть аплинка лол

littleboobs commented:
Закрыл обсуждение.

Меня утомило, что на каждое изменение появляется 10 зондбе или просто серунов, которые выстрачивают на гитхабе своё важное мнение. Добро пожаловать в дискорд, множество каналов, где вы можете изложить свои мысли и возможно их кто-то прочитает и даже ответит. Если это продолжится дальше, но будем обсуждать бан срунов из обсуждения на гитхабе. Спасибо за внимание! 🙏

vovakons committed:
reduce holo cigare price to 20 tk

Понижение цены голосигары с 35 до 20 тк по просьбе меланхолика.

dageavtobusnick commented:
Слишком сильный предмет по итогу получается. Убери обнуление разброса. Можешь добавить снижение, но не обнуление точно

vovakons committed:
holo cigar spread balance

Трейт от голосигары теперь не убирает разброс, теперь уменьшает разброс
в два раза.

vovakons commented:
> Слишком сильный предмет по итогу получается. Убери обнуление разброса. Можешь добавить снижение, но не обнуление точно

Сделано 👍🏻

littleboobs committed:
Merge branch 'master220' into holosigare-fix

dageavtobusnick committed:
Merge branch 'master220' into holosigare-fix
